analyses/OR_cd_2010/03_sim_OR_cd_2010.R

```diff
@@ -0,0 +1,33 @@
+###############################################################################
+# Simulate plans for `OR_cd_2010`
+# © ALARM Project, October 2022
+###############################################################################
+
+# Run the simulation -----
+cli_process_start("Running simulations for {.pkg OR_cd_2010}")
+
+set.seed(2010)
+plans <- redist_smc(map, nsims = 5000, runs = 2L, counties = county) %>%
```

Comment:
Sorry one last thing, it looks like you didn't replace `county` with `pseudo_county`, but create and mention it in `02` and `04`, respectively.

Reply:
Issue should be fixed--`pseudo counties` didn't have any major change in the muni or county splits, so I found that using just `county` was okay.

Reply:
Thanks, that seems reasonable, as the current county and muni splits were pretty solid.